app/controllers/course/achievement/condition/achievements_controller.rb

```diff
@@ -0,0 +1,4 @@
+class Course::Achievement::Condition::AchievementsController <
+  Course::Condition::AchievementsController
+  include Course::AchievementConditionalConcern
```

Comment:
I don't think so because both `Course::Achievement::Condition::AchievementsController` and `Course::Mission::Condition::AchievementsController` inherit from `Course::Condition::AchievementsController` but they have different `set_conditional`. The former needs to find an achievement while the latter needs to look up a mission. Or am I missing smth?